cmd/clouddriver/clouddriver.go

```diff
@@ -67,6 +67,7 @@ func init() {
 	}
 
 	arcadeClient.WithAPIKey(arcadeAPIKey)
+	arcadeClient.WithShortExpiration(60)
```

Comment:
Magic number
I think replacing 60 with a constant might make it easier to understand